migrate/migrations/20240918060355-account-migrations.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    return queryInterface.createTable('account', {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+        allowNull: false,
```

Comment:
nit: can check what is default value of `allowNull` ? I think it is dependent on the type of field, like for primary key, createdAt fields it is 'false' by default. 
Can be checked and this checks can be removed wherever it is unnecessary.